README.md

```diff
@@ -1,15 +1,17 @@
+# Nomic
+
+## Player scores
+
 player      | order | points
 ----------- | ----- | ------
 akvadrako   | Ba    | -2
 RamonRB     | Be    | 11
 rknegjens   | K     | 0
 roarandrock | W     | 0
 
-## New Rules
-
-there is nothing here
+## Original rules
```

Comment:
Again we don't need two levels of headings.

Reply:
the point is that Mutable/Immutable and New/Original are separate things. I propose removing the latter then